README.md

```diff
@@ -1 +1,76 @@
-# exercise-use-gitignore
\ No newline at end of file
+# Welcome to the Use .gitignore exercise!
+
+This is an exercise to check your knowledge on using a `.gitignore` file. It is automatically graded via a workflow once you have completed the instructions.
+
+## About this exercise
+
+:warning: A grading script exists under `.github/workflows/grading.yml`. You do not need to use this workflow for any purpose and **altering its contents will affect the repository's ability to assess your exercise and give feedback.**
+
+:warning: This exercise utilizes [GitHub Actions](https://docs.github.com/en/actions), which is free for public repositories and self-hosted runners, but may incur charges on private repositories. See _[About billing for GitHub Actions]_ to learn more.
+
+:information_source: The use of GitHub Actions also means that it may take the grading workflow a few seconds and sometimes minutes to run.
+
+## Instructions
+
+<!-- Specific instructions for your exercise -->
+
+Please complete the instructions below:
+
+1. Create your own copy of this repository by using the [Use this template](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template#creating-a-repository-from-a-template) button.
+2. Edit the `.gitignore` file to the root of the repository
+3. The `.gitignore` file should meet the following conditions in addition to what already exists in the file:
+   - Ignores all files starting with the letter `z`
+   - Ignores a file call `.env`
+   - Ignores a top-level directory named `artifacts`
```

Comment:
```suggestion
2. Edit the `.gitignore` file so that it meets the following conditions:
   - Ignores all files starting with the letter `z`.
   - Ignores a file called `.env`.
   - Ignores a top-level directory named `artifacts`.
```